controls/V-73759.rb

```diff
@@ -78,56 +78,54 @@
+  #   describe 'This system is dedicated to the management of Active Directory, therefore this system is exempt from this control' do
+  #     skip 'This system is dedicated to the management of Active Directory, therefore this system is exempt from this control'
+  else
+    describe security_policy do
+      its('SeDenyNetworkLogonRight') { should include 'S-1-5-32-546' }
+    end
+    if domain_role == '3'
+      domain_admin_sid_query = <<-EOH
+        $group = New-Object System.Security.Principal.NTAccount('Domain Admins')
+        $sid = $group.Translate([security.principal.securityidentifier]).value
+        $sid | ConvertTo-Json
+      EOH
+      domain_admin_sid = json(command: domain_admin_sid_query).params
+      
+      enterprise_admin_sid_query = <<-EOH
+        $group = New-Object System.Security.Principal.NTAccount('Enterprise Admins')
+        $sid = $group.Translate([security.principal.securityidentifier]).value
+        $sid | ConvertTo-Json
+      EOH
+      enterprise_admin_sid = json(command: enterprise_admin_sid_query).params
+
+      describe security_policy do
+        its('SeDenyNetworkLogonRight') { should include "#{domain_admin_sid}" }
       end
-    else
-      if is_AD_only_system
-        impact 0.0
-        desc 'This system is dedicated to the management of Active Directory, therefore this system is exempt from this control'
-        describe 'This system is dedicated to the management of Active Directory, therefore this system is exempt from this control' do
-          skip 'This system is dedicated to the management of Active Directory, therefore this system is exempt from this control'
-        end
-      else
-        domain_admin_sid_query = <<-EOH
-          $group = New-Object System.Security.Principal.NTAccount('Domain Admins')
-          $sid = $group.Translate([security.principal.securityidentifier]).value
-          $sid | ConvertTo-Json
-        EOH
-        domain_admin_sid = json(command: domain_admin_sid_query).params
-        
-        enterprise_admin_sid_query = <<-EOH
-          $group = New-Object System.Security.Principal.NTAccount('Enterprise Admins')
-          $sid = $group.Translate([security.principal.securityidentifier]).value
-          $sid | ConvertTo-Json
-        EOH
-        enterprise_admin_sid = json(command: enterprise_admin_sid_query).params
- 
+      describe security_policy do
+        its('SeDenyNetworkLogonRight') { should include "#{enterprise_admin_sid}" }
+      end
+
+      describe.one do
         describe security_policy do
-          its('SeDenyNetworkLogonRight') { should include "#{domain_admin_sid}" }
+            its('SeDenyNetworkLogonRight') { should include "S-1-5-113" }
         end
         describe security_policy do
-          its('SeDenyNetworkLogonRight') { should include "#{enterprise_admin_sid}" }
+            its('SeDenyNetworkLogonRight') { should include "S-1-5-114" }
```

Comment:
Added the SIDs for local accounts as per the updated STIG guidance. Test pending.

Reply:
Tested after adding Local accounts where needed